packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -469,7 +469,9 @@ export default class Meeting extends StatelessWebexPlugin {
   resourceId: any;
   resourceUrl: string;
   selfId: string;
-  state: any;
+  state: any;  
+  isMultistream: boolean;
```

Comment:
I think you meant to do this change on a branch from "beta" branch and merge this PR into "beta" branch, not master. We don't have the isMultistream parameter on SDK master